content/docs/command-reference/add.md

```diff
@@ -374,33 +376,33 @@ outs:
     path: data.xml
 ```
 
+[link]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+[external cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
+
 ## Example: Transfer to remote storage
 
 When you have a large dataset in an external location, you may want to track it
-as if it was in your project, but without downloading it locally (for now). The
-`--to-remote` option lets you do so, while storing a copy
-[remotely](/doc/command-reference/remote) so it can be
-[pulled](/doc/command-reference/plots) later.
+in your <abbr>project</abbr> without downloading it (yet), for example if
+there's not enough space in your current environment.
+
+The `--to-remote` option lets you bootstrap the data locally by creating a
+`.dvc` file, while storing a copy [remotely](/doc/command-reference/remote).
```

Comment:
When I read "storing a copy," it makes me think that the data might still be stored locally, and then copied to the remote. What about "storing the data"?

Reply:
Removed "storing a copy". Went for "... transfers it directly to remote storage..."